test/run_test.py

```diff
@@ -838,8 +843,23 @@ def parse_args():
         nargs='?',
         type=str,
         const=SPECIFIED_TEST_CASES_FILE,
-        help='runs specified test cases from previous OSS CI stats from a file, format CSV',
-
+        help='load specified test cases file dumped from previous OSS CI stats, format CSV. '
+        ' If all test cases should run for a <test_module> please add a single row: \n'
+        ' test_filename,test_case_name\n'
+        ' ...\n'
+        ' <test_module>,__all__\n'
+        ' ...\n'
+        'file content is used based on option "--use-specified-test-cases-for".'
```

Comment:
```suggestion
        'how we use the stats will be based on option "--use-specified-test-cases-by".'
```
I think this could help clarify